lib/build.js

```diff
@@ -338,6 +343,11 @@ export default {
     targetName = targetName || this.activeTarget[p];
 
     Promise.resolve(this.targets[p]).then(targets => {
+      if (atom.packages.isPackageActive('linter')) {
```

Comment:
It's not about giving `linterRegistry`, it's about minimizing usage of external API